catalog/ui/catalog-ui-search/src/main/webapp/component/session-timeout-modal/session-timeout.hbs

```diff
@@ -0,0 +1,18 @@
+{{!--
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+ --}}
+<div id="message">
+    <div>Your session will expire in <label id="timer">{{time}}</label> seconds.
+    Press "Continue working" if you need more time.</div>
```

Comment:
I feel like it's more common to say something along the lines of 
"Press 'Continue working' to remain logged in."